novawallet/Modules/NetworkManagement/KnownNetworksList/KnownNetworksListViewFactory.swift

```diff
@@ -2,29 +2,32 @@ import Foundation
 import SoraFoundation
 
 struct KnownNetworksListViewFactory {
-    static func createView() -> KnownNetworksListViewProtocol? {
+    static func createView(
+        successAddPresenting: (wireframe: ModalAlertPresenting, view: ControllerBackedProtocol)
```

Comment:
I thought we will improve the logic in all places not only in the CustomNetwork module - don't see a need to move ModalPresenting stuff around. Probably, let's do it in a separate PR.

Reply:
Damn, I was sure I removed it everywhere else. Thanks, I will remove it in separate PR for sure.